### ISSUE TITLE
Resources corrupted

### ISSUE THREAD

**Issue by DarkCitia** [AyJxx/WaterSystem#1]
picture files are corrupted that can’t be loaded.
the showcase is very good , but without boat model I think I can't set the boat controller correctly.

**Comment by AyJxx** [comment]
Yes, repository here is corrupted. You can find it here https://bitbucket.org/AyJxx/watersystem/src/master/
